ytypes/gnmi_test.go

```diff
@@ -496,6 +498,141 @@ func TestUnmarshalSetRequest(t *testing.T) {
 			}},
 		},
 		wantErr: true,
+	}, {
+		desc: "mix of an error and a non-error update with best-effort flag",
```

Comment:
Actually I was thinking of putting all the errors into a single SetRequest. If you just add another test case with everything in these three cases or just merge all of these into one then this LGTM.

Reply:
Moving them all into one.